modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/get-in-touch/index.js

```diff
@@ -15,13 +11,19 @@ btnCall.onclick = function(){
 	window.location.href = 'tel:' + valueCall;
 }
 
-for(let i = 0; i < infoCookie.length; i++) {
-	auxCookie = infoCookie[i];
-	while (auxCookie.charAt(0) == ' ') {
-		auxCookie = auxCookie.substring(1);
+function getCookie(name) {
+	name = name + '=';
+	const decodedCookie = decodeURIComponent(document.cookie);
+	const cookies = decodedCookie.split(';');
+	for (let i = 0; i < cookies.length; i++) {
+		const cookie = cookies[i].trim();
+		if (cookie.indexOf(name) == 0) {
+		return cookie.substring(name.length, cookie.length);
+		}
 	}
-	if (auxCookie.indexOf(nameCookie) == 0) {
-		applicationId = auxCookie.substring(nameCookie.length, auxCookie.length);
-		document.getElementById("content-agent-text-your-application").textContent = "Your Application #"+applicationId;
-	}
-}
\ No newline at end of file
+}
+
+if(getCookie(nameCookie) != undefined && getCookie(nameCookie) != "" )
```

Comment:
this line could be replaced just for `if (getCookie(nameCookie)) {}`

Any string inside this variable or function return, is already true

Reply:
Also, you are using `getCookie` 3 times, you could assign to one variable, and use it